### ISSUE TITLE
`--camel-case` option no longer works

### ISSUE THREAD

**Issue by ctrlplusb** [RobinBlomberg/kysely-codegen#43]
I upgraded from `0.6.0` to `0.7.4` and it now appears that the `--camel-case` option no longer works.

```
kysely-codegen --camel-case --dialect mysql --out-file ./core/db/db.types.ts
```

**Comment by ctrlplusb** [comment]
Rolling back to `0.6.2` appears to fix this. It looks to be a regression in `0.7.x`.

**Comment by RobinBlomberg** [comment]
Should be fixed in `kysely-codegen@0.8.0` now!
